packages/web/src/components/edit/fields/price-and-audience/priceAndAudienceSchema.ts

```diff
@@ -0,0 +1,152 @@
+import { USDCPurchaseConfig } from '@audius/common/hooks'
+import {
+  isContentUSDCPurchaseGated,
+  StreamTrackAvailabilityType,
+  AccessConditions
+} from '@audius/common/models'
+import { Nullable, formatPrice } from '@audius/common/utils'
+import { z } from 'zod'
+
+import {
+  ALBUM_TRACK_PRICE,
+  AccessAndSaleFormValues,
+  PREVIEW,
+  PRICE,
+  STREAM_AVAILABILITY_TYPE,
+  STREAM_CONDITIONS
+} from '../types'
+
```

Comment:
assuming this is just copied over from PriceAndAudienceField

Reply:
yeah exactly